docs/reference/announcements-release-notes/880/880-release-notes.md

```diff
@@ -571,6 +563,15 @@ As well as bearer token and client credentials authentication, you can now confi
 
 <p><a href="../../../../self-managed/components/modeler/web-modeler/configuration/#available-authentication-methods" class="link-arrow">Available authentication methods</a></p>
 
+### Element template editor onboarding
+
+<!-- https://github.com/camunda/product-hub/issues/3021 -->
+
+Web Modeler now provides a low-coder friendly UX for creating building blocks.
+Whether you create an element template from scratch or from a task, you can quickly set and narrow down the properties so process developers using these templates can quickly and confidently wire building blocks together into an E2E process.
```

Comment:
I basically copied the release notes from https://github.com/camunda/product-hub/issues/3021, but maybe there's still room for improvement. They don't really mention the new creation flow that lets you create templates for different tasks types more easily (see https://docs.camunda.io/docs/next/components/modeler/web-modeler/element-templates/element-template-generator/#generate-a-bpmn-element-template).

Reply:
cc @bastiankoerber could you have a look at this?